sql/src/main/java/io/crate/execution/ddl/tables/AlterTableOperation.java

```diff
@@ -202,6 +261,220 @@ private Session session() {
         return result;
     }
 
+    private CompletableFuture<Long> executeAlterTableChangeNumberOfShards(AlterTableAnalyzedStatement analysis) {
+        final DocTableInfo table = analysis.table();
+        final boolean isPartitioned = table.isPartitioned();
+        String sourceIndexName, sourceIndexAlias;
```

Comment:
haven't seen this in a while :), although it's ok syntactically, we don't generally define multiple variables in one go

Reply:
what's wrong with that?  :)

I'll change it on master after merge - promise

Reply:
If we don't want to have multiple variable definitions in a single line we should see if there is a checkstyle rule for that and enable it.